test/unit/useManualQuery.test.js

```diff
@@ -0,0 +1,23 @@
+import { useManualQuery, useClientRequest } from '../../src';
+
+jest.mock('../../src/useClientRequest');
+
+const TEST_QUERY = `query Test($limit: Int) {
+  tests(limit: $limit) {
+    id
+  }
+}`;
+
+describe('useManualQuery', () => {
+  afterEach(() => {
+    jest.unmock('../../src/useClientRequest');
```

Comment:
Should the `mock` be in a `beforeEach`? I'm unsure on the inner workings of jest but it reads like after the first test is run in this file then the imported file is unmocked for the remaining ones.

Reply:
I think this was a misunderstanding from myself on how the mocks work, removing it has no impact. Will remove.